library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -183,7 +184,7 @@ public class ChuckerInterceptor private constructor(
     private fun createTempTransactionFile(): File? {
         val cache = cacheDirectoryProvider.provide()
         return if (cache == null) {
-            IOException("Failed to obtain a valid cache directory for Chucker transaction file").printStackTrace()
+            Logger.warn("Failed to obtain a valid cache directory transaction file")
```

Comment:
Should we generate an `Exception` here and call `Logger.error` instead?

Reply:
I removed `Exception` because I thought it was used only to avoid `println` as there was no access here to a logger. I'm fine either way and can bring it back if you think it helps.

Reply:
I'm unsure about the value added tbh. I'm fine either way 👍

Reply:
Ok, I'll leave it them. Just will fix a typo that I just noticed.